scripts/analytics/base.py

```diff
@@ -9,10 +9,13 @@
 from website.app import init_app
 from website.settings import KEEN as keen_settings
 from keen.client import KeenClient
+from scripts import utils as script_utils
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('scripts.analytics')
 logging.basicConfig(level=logging.INFO)
 
+script_utils.add_file_logger(logger, 'keen_analytics')
```

Comment:
Nit: We should generally avoid import side-effects, i.e. importing a module shouldn't create file. So the file logger should be created in the `run_main` function for the scripts being run.